fvm/src/call_manager/state_access_tracker.rs

```diff
@@ -0,0 +1,260 @@
+// Copyright 2021-2023 Protocol Labs
+// SPDX-License-Identifier: Apache-2.0, MIT
+use std::cell::RefCell;
+use std::iter;
+
+use anyhow::Context;
+use fvm_shared::address::{Address, Protocol};
+use fvm_shared::ActorID;
+
+use crate::gas::{GasTimer, GasTracker, PriceList};
+use crate::history_map::HistoryMap;
+use crate::kernel::{ClassifyResult, Result};
+
+struct StateAccessLayer {
+    addresses_height: usize,
+    actors_height: usize,
+}
+
+pub struct StateAccessTracker {
+    price_list: &'static PriceList,
+    actors: RefCell<HistoryMap<ActorID, bool>>,
```

Comment:
Comment explaining this `bool` value please (false => read-but-not-written, true => written)

Reply:
I'm just going to use an enum.